md-docs/user_guide/monitoring/drift_explainability.md

```diff
@@ -0,0 +1,37 @@
+# Drift Explainability
+
+[Monitoring]  is a crucial aspect of the machine learning lifecycle, as it enables tracking the model's performance and its data over time,
+ensuring the model continues to function as expected. However, monitoring only is not enough when it comes to the adaptation phase.
+
+In order to make the right decisions, you need to understand what were the main factors that led to the drift in the first place, so that
+the correct actions can be taken to mitigate it.
+
+The MLCube Platform supports this process by offering what we refer to as "**Drift Explainability Reports**", 
+automatically generated upon the detection of a drift and containing several elements that should help you diagnose the root causes 
+of the change occurred.
+
+You can access the reports by navigating to the `Drift Explainability` tab in the sidebar of the task page.
+
+## Structure
+
+A Drift Explainability Report consists in comparing the reference data and the portion of production data where the drift was identified, hence 
+those belonging to the new concept. Notice that these reports are generated after a sufficient amount of samples has been collected after the drift.
+If the distribution moves back to the reference before enough samples are collected, the report might not be generated.
```

Comment:
That's why elements componing the report needs a given amount of data to guarantee statistical reliability of the shown results. (sicuramente la si può scrivere meglio)